libraries/AP_NavEKF3/AP_NavEKF3.h

```diff
@@ -453,6 +458,10 @@ class NavEKF3 {
     AP_Int8 _gsfResetMaxCount;      // maximum number of times the EKF3 is allowed to reset it's yaw to the EKF-GSF estimate
     AP_Float _err_thresh;           // lanes have to be consistently better than the primary by at least this threshold to reduce their overall relativeCoreError
     AP_Int32 _affinity;             // bitmask of sensor affinity options
+    AP_Float _dragObsNoise;         // drag specific force observatoin noise (m/s/s)**2
+    AP_Float _ballisticCoef_x;      // ballistic coefficient measured for flow in X body frame directions
```

Comment:
these parameter names should probably also get _drag prefixed.

Reply:
Disagree.